brooklyn-tosca-transformer/src/test/resources/templates/script1.tosca.yaml

```diff
@@ -19,10 +19,10 @@ topology_template:
       - host: a_server
       interfaces:
         Standard:
-           create: https://raw.githubusercontent.com/ahgittin/tosca-demo/master/script1/scripts/create.sh
-           configure: https://raw.githubusercontent.com/ahgittin/tosca-demo/master/script1/scripts/configure.sh
-           start: https://raw.githubusercontent.com/ahgittin/tosca-demo/master/script1/scripts/start.sh
-           stop: https://raw.githubusercontent.com/ahgittin/tosca-demo/master/script1/scripts/stop.sh
+           create: classpath:///artifacts/lifecycle/softwarecomponent/create.sh
```

Comment:
should these be namespaced as `classpath:///brooklyn/tosca/test/templates/script1/*.sh` or similar?  as it stands the paths are too vague.